2023/ideas-list.md

```diff
@@ -27,5 +27,6 @@ page of each organization under the NumFOCUS umbrella at this page.
 - [SciML](https://sciml.ai/dev/#google_summer_of_code)
 - [Taskflow](https://github.com/taskflow/GSoC2023)
 - [TNL](https://gitlab.com/tnl-project/tnl/-/wikis/GSoC-2023)
+- [Open Science Labs](https://hackmd.io/@GvoPVECJQAmNl6JmM2UPMQ/BJEITVBAj)
 
```

Comment:
@xmnlab please organize it according to the sorted order used

Reply:
oh that is right .. thanks for point that .. working on that right now.

Reply:
@henrykironde I think that it is fixed now. thank you for your review.